Source/ui/UI.js

```diff
@@ -33,8 +33,13 @@ export function message(text, document) {
  * by pressing the OK button.
  */
 export function alert(title, text) {
-  const app = NSApplication.sharedApplication()
-  app.displayDialog_withTitle(text, title)
+  const dialog = NSAlert.alloc().init()
```

Comment:
BTW - in Obj-C you can do `[class new]` as a shortcut to `[[class alloc] init]`. It doesn't matter, but I expect you could do `NSAlert.new()` here.

Reply:
indeed but I find it confusing with the `new` keyword in javascript :)

Reply:
Fair enough :)